EventProcessor/EventProcessor.WebJob/app.config

```diff
@@ -88,6 +89,18 @@
     </extensions>
   </system.serviceModel>
   <appSettings>
-    <add key="Microsoft.ServiceBus.ConnectionString" value="Endpoint=sb://[your namespace].servicebus.windows.net;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=[your secret]" />
+    <add key="docdb.DatabaseId" value="DevMgmtDB" />
+    <add key="docdb.DocumentCollectionId" value="DevMgmtCollection" />
+    <add key="DeviceRulesStoreContainerName" value="devicerules" />
+    <add key="DeviceRulesTableName" value="DeviceRulesNormalizedTable" />
+    <add key="ActionMappingStoreContainerName" value="actionmappings" />
+    <add key="ActionMappingStoreBlobName" value="mappings.json" />
+    <add key="AsaRefDataRulesBlobName" value="devicerules.json" />
+    <add key="SolutionName" value="Remote Monitoring Solution" />
+    <add key="TelemetryStoreContainerName" value="devicetelemetry" />
+    <add key="TelemetryDataPrefix" value="devicetelemetry/" />
+    <add key="TelemetrySummaryPrefix" value="devicetelemetry-summary/" />
+    <add key="AlertsStoreContainerName" value="rulesoutput" />
+    <add key="DeviceAlertsDataPrefix" value="" />
```

Comment:
Running the current version locally, the EventProcessor.WebJob fails with an exception due to the following missing keys:
* `eventHub.HubName`
* `eventHub.ConnectionString`
* `eventHub.StorageConnectionString`